app/controllers/download_controller.rb

```diff
@@ -195,13 +195,9 @@ def set_distro_flavor distro
     end
   end
 
-  def set_flavours
-    if @data.nil?
-      head :forbidden
-    else
-      # collect distro types from @data
-      @flavors = @data.values.collect { |i| i[:flavor] }.uniq.sort { |x, y| x.downcase <=> y.downcase }
-    end
+  def set_flavors
+    return head :forbidden unless @data
+    @flavors = @data.values.collect { |i| i[:flavor] }.uniq.sort_by(&:downcase)
```

Comment:
The `sort` now is not only shorter, also more efficient. I would add it to the commit message

Reply:
Updated the commit description, thanks!